app/assets/stylesheets/nfg_ui/network_for_good/admin/nfg_theme/custom/_sortable.scss

```diff
@@ -0,0 +1,29 @@
+// Shared sorting styles
+.slat, .option {
+  &.ui-sortable-placeholder {
+    visibility: visible !important;
+    background: theme-color("light");
+    border: $border-width dashed $border-color;
+  }
+  &.ui-sortable-helper { opacity: .5; }
+}
+// Slat
+.slat {
+  &.ui-sortable-placeholder {
+    + .slat { border-top: none; }
+    + &.ui-sortable-helper {
+      border-top-style: dashed;
+      + &.slat { border-top: none; }
+    }
+  }
+  &.ui-sortable-helper {
+    border-top: none;
+    border-top: none;
+    + &.ui-sortable-placeholder { border-top: $border-width dashed $border-color; }
+  }
+}
+
+// Sortable options (used on form fields for checkboxes/radios/select options)
+.option {
+  &.ui-sortable-placeholder { margin-bottom: ($spacer * .25); }
+}
```

Comment:
@kylejmorin if sortable has a pattern for implementation, display app may benefit from showing how it's implemented. Not sure how widely used sorting is.

Reply:
I don't think it's used widely enough, but if it becomes then we should add it. I also think the JS is not in the nfg_ui gem.